etc/answers-core.api.md

```diff
@@ -136,17 +136,14 @@ export enum Direction {
 // @public
 export interface DisplayableFacet extends Facet {
     displayName: string;
-    fieldId: string;

```

Comment:
We decided to include properties from interfaces which are being extended in order to provide slightly more readable documentation.  Is there a need to remove the duplicated props here and in DisplayableFacetOption?

Reply:
I see. I noticed the duplications and decided to remove it, didn't realize it was intentional for documentation purpose. I will add it back in!